exp-player/addon/components/exp-rating-form/component.js

```diff
@@ -4,569 +4,580 @@ import layout from './template';
+    }),
+    generateSchema({
+        question: 'measures.questions.14.label',
+        type: 'radio',
+        page: 6,
+        items: items['14'],
+        scale: [
+            'measures.questions.14.options.disagreeStrongly',
+            'measures.questions.14.options.disagree',
+            'measures.questions.14.options.neutral',
+            'measures.questions.14.options.agree',
+            'measures.questions.14.options.agreeStrongly'
+        ],
+        options: {labelTop: true}
+    }),
+    generateSchema({
+        question: 'measures.questions.15.label',
+        type: 'radio',
+        page: 6,
+        items: items['15'],
+        scale: [
+            'measures.questions.15.options.disagreeStrongly',
+            'measures.questions.15.options.disagree',
+            'measures.questions.15.options.neutral',
+            'measures.questions.15.options.agree',
+            'measures.questions.15.options.agreeStrongly'
+        ],
+        options: {labelTop: true}
+    }),
+    generateSchema({
+        question: 'measures.questions.16.label',
+        type: 'radio',
+        page: 7,
+        items: items['16'],
+        scale: [
+            'measures.questions.16.options.notAtAll',
+            'measures.questions.16.options.aLittle',
+            'measures.questions.16.options.quiteaBit',
+            'measures.questions.16.options.completely'
+        ],
+        options: {labelTop: true}
+    }),
+    generateSchema({
+        question: 'measures.questions.17.label',
+        type: 'radio',
+        page: 7,
+        items: items['17'],
+        scale: [
+            'measures.questions.17.options.disagreeStrongly',
+            'measures.questions.17.options.disagree',
+            'measures.questions.17.options.neutral',
+            'measures.questions.17.options.agree',
+            'measures.questions.17.options.agreeStrongly'
+        ],
+        options: {labelTop: true}
+    })
 ];
 
 const Validations = buildValidations(generateValidators(questions));
 
 export default ExpFrameBaseComponent.extend(Validations, {
     type: 'exp-rating-form',
     layout: layout,
-    questions: Ember.computed(function() {
+    currentPage: 1,
+    totalPages: 7,
+    progressBarPage: Ember.computed('currentPage', function () {
+        return this.currentPage + 4;
+    }),
+    questions: Ember.computed(function () {
         var condition = this.get('session').get('experimentCondition');
         if (condition === '7pm') {
-            questions[3]['question'] = 'measures.questions.4.label.7pm';
+            questions[2]['question'] = 'measures.questions.3.label.7pm';
         } else if (condition === '10am') {
-            questions[3]['question'] = 'measures.questions.4.label.10am';
+            questions[2]['question'] = 'measures.questions.3.label.10am';
         }
         return questions;
     }),
-    responses: Ember.computed('questions', function() {
+    responses: Ember.computed('currentPage', function () {
```

Comment:
It may make more sense to have this computed not subscribe to any attributes but to also use the [`volatile` flag](http://emberjs.com/api/classes/Ember.ComputedProperty.html#method_volatile)